models/hospital.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {}
+  Hospital.init(
+    {
+      id: {
+        field: "hospital_uuid",
+        type: DataTypes.UUID,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      hospitalName: {
+        field: "hospitalname",
+        type: DataTypes.STRING,
+        unique: true,
+        allowNull: false,
+      },
+      recordUpdateSource: {
+        field: "recordupdatesource",
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      recordUpdateTimestamp: {
+        field: "recordupdatetimestamp",
+        type: DataTypes.DATE,
+      },
+      recordCreateSource: {
+        field: "recordcreatesource",
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      recordCreateTimestamp: {
+        field: "recordcreatetimestamp",
+        type: DataTypes.DATE,
+      },
+    },
+    {
+      sequelize,
+      timestamps: true,
+      createdAt: "recordCreateTimestamp",
+      updatedAt: "recordUpdateTimestamp",
```

Comment:
And then we'll customize Sequelize to set them automatically...

Reply:
I see, you're providing the Model property here, not the database column name.